cli/main.rs

```diff
@@ -105,6 +104,7 @@ pub fn print_file_info(
     &worker.state,
     module_specifier,
   ).and_then(move |out| {
+    // TODO(bartlomieju): it will show filename of compiled module (if it is TS)
```

Comment:
This method should have almost identical body as `fetch_module_meta_data_and_maybe_compile_async`